src/pages/Onboarding/Onboarding.scss

```diff
@@ -71,9 +74,7 @@
   }
 
   main {
-    padding-top: 64px;
-    position: absolute;
-    bottom: 60px;
+    position: static;
```

Comment:
`static` is a default value, is it necessary?

Reply:
no, it will removed